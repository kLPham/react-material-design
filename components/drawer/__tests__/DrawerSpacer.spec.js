import React from 'react';
import { shallow } from 'enzyme';
import DrawerSpacer from '../drawerSpacer';

describe('<DrawerSpacer />', () => {
    it('should render', () => {
        const drawerSpacer = shallow(
            <DrawerSpacer />,
    );
        expect(drawerSpacer).toMatchSnapshot();
    });
});
