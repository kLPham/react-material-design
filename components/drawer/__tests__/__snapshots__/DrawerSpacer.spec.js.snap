// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DrawerSpacer /> should render 1`] = `
<div
  className="mdc-temporary-drawer__toolbar-spacer"
/>
`;
